app/src/browser/system-tray-manager.es6

```diff
@@ -78,7 +78,16 @@ class SystemTrayManager {
 
   _onClick = () => {
     if (this._platform !== 'darwin') {
-      this._application.emit('application:show-main-window');
+	  if(this._application.windowManager.getVisibleWindowCount() == 0) {
+	      this._application.emit('application:show-main-window');
+	  }
+	  else {
+		const visibleWindows = this._application.windowManager.getVisibleWindows();
+		Object.keys(visibleWindows).forEach(key => {
```

Comment:
It looks like `getVisibleWindows` is returning an array but this is treating it as an object? I think this should be `visibleWindows.forEach(win => win.hide())`?

Reply:
Ok, corrected.